hack/tools/go.mod

```diff
@@ -1,13 +1,13 @@
 module k8s.io/kubernetes/hack/tools
 
-go 1.19
+go 1.21
```

Comment:
hmm @liggitt probably we should be forcing this to stay in sync with the root go.mod?